gitclient/datadog_checks/gitclient/data/conf.yaml.example

```diff
@@ -0,0 +1,57 @@
+## All options defined here are available to all instances.
+#
+init_config:
+
+    ## @param service - string - optional
+    ## Attach the tag `service:<SERVICE>` to every metric, event, and service check emitted by this integration.
+    ##
+    ## Additionally, this sets the default `service` for every log source.
+    #
+    # service: <SERVICE>
+
+## Every instance is scheduled independent of the others.
+#
+instances:
+
+    ## @param git_http_repo_url - string - required
```

Comment:
These need to go in `gitclient/assets/configuration/spec.yaml` then you run `ddev validate config --sync gitclient` to generate this file. You can [check apache core integration](https://github.com/DataDog/integrations-core/blob/master/apache/assets/configuration/spec.yaml) for how to edit the spec file: